api/v1beta1/temporalcluster_types.go

```diff
@@ -799,6 +799,11 @@ func (s *ClusterArchivalSpec) IsEnabled() bool {
 	return s != nil && s.Enabled
 }
 
+type ClusterReplicationSpec struct {
+	EnableGlobalNamespace  bool  `json:"enableGlobalNamespace"`
```

Comment:
Please make this field optionnal and define a default value (what's the temporal default value) ?